pkg/manager/impl/version_manager_test.go

```diff
@@ -0,0 +1,15 @@
+package impl
+
+import (
+	"context"
+	"testing"
+
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVersionManager_GetVersion(t *testing.T) {
+	versionManager := NewVersionManager()
+	_, err := versionManager.GetVersion(context.Background(), &admin.GetVersionRequest{})
```

Comment:
nit: can we set the build, version, etc in the VersionManager and check that it does get passed in the response obj?